### PR TITLE
E2e test fixes

### PR DESCRIPTION
**Pull request by MoonShiesty**
This PR cleans up e2e tests, follow-up to @siosw PR#257

### CODE REVIEW

File: contracts/scripts/e2e/utils.ts [hunk 1]
```diff
@@ -1,76 +1,76 @@
-const l2Provider = new ethers.providers.JsonRpcProvider(
+import * as hardhat from "hardhat"
+import { l1OracleAddress, l2PortalAddress, l2StandardBridgeAddress } from "./addresses";
+
+const l2Provider = new hardhat.ethers.providers.JsonRpcProvider(
   "http://localhost:4011"
 );
 
-const l1Provider = new ethers.providers.JsonRpcProvider(
+const l1Provider = new hardhat.ethers.providers.JsonRpcProvider(
```
Comment (chalkandpaste) [inline contracts/scripts/e2e/utils.ts:8]: Why the switch from "ethers" to "hardhat"? I see we continue to use "ethers", so I'm guessing there's a functionality reason

File: contracts/scripts/e2e/bridge/test_standard_bridge_withdraw_eth.ts [hunk 2]
```diff
@@ -47,25 +44,28 @@ async function main() {
...
-    async (batchNumber, previousInboxSize, inboxSize, event) => {
-      const tx = await event.getTransaction();
-      lastConfirmedBlockNumber = await getLastBlockNumber(tx.data);
+    (event) => {
+      console.log(`TxBatchAppended blockNum: ${event.blockNumber}`)
     }
   );
 
-  rollup.on(rollup.filters.AssertionConfirmed(), (id) => {
+  rollup.on(rollup.filters.AssertionConfirmed(), async (id: BigNumber) => {
```
Comment (chalkandpaste) [inline contracts/scripts/e2e/bridge/test_standard_bridge_withdraw_eth.ts:58]: I see this is a copypasta with L129-133 of `contracts/scripts/e2e/bridge/test_standard_bridge_erc20.ts`. Is there a good way of cleaning this up?

**Review by chalkandpaste — APPROVED** [review]
LGTM, except I have a suspicion about the removal of `onlyProxy` modifier in the contracts. Would love to see that resolved, but if no one else has an issue, I'm not going to stop this from going through

File: contracts/scripts/e2e/utils.ts [hunk 3]
```diff
@@ -1,76 +1,76 @@
-const l2Provider = new ethers.providers.JsonRpcProvider(
+import * as hardhat from "hardhat"
```
Comment (LEAFERx) [inline contracts/scripts/e2e/utils.ts:1]: can do `import { ethers } from "hardhat"`

**Review by LEAFERx — APPROVED** [review]
lgtm

File: contracts/scripts/e2e/bridge/test_standard_bridge_withdraw_eth.ts (hunk 2 above)
Comment (MoonShiesty) [reply contracts/scripts/e2e/bridge/test_standard_bridge_withdraw_eth.ts:58]: I think once we have all the `e2e` tests passing, there's a lot of cleanup and refactoring we could do

**Comment by MoonShiesty** [comment]
@chalkandpaste rebased and ready for review, this PR is now mostly just typescript fixes I found while trying to debug the e2e tests

**Review by chalkandpaste — APPROVED** [review]

**Pull request merged**